cloud/src/recycler/checker.cpp

```diff
@@ -437,13 +439,17 @@ int InstanceChecker::init_storage_vault_accessors(const InstanceInfoPB& instance
         }
 
         if (vault.has_hdfs_info()) {
+#ifdef UNIT_TEST
+            auto accessor = std::make_shared<MockAccessor>();
```

Comment:
use injection instead of MACRO 

e.g.
```
auto create_hdfs_accessor(...) {
    TEST_SYNC_POINT_RETURN_WITH_VALUE("create_hdfs_accessor for UT", ...);
    return std::make_shared<HdfsAccessor>(vault.hdfs_info());
}



...
if (vault.has_hdfs_info()) {
    ....
    auto accessor create_hdfs_accessor(vault.hdfs_info());
....
...
```